int64validator/at_least_sum_of.go

```diff
@@ -0,0 +1,106 @@
+package int64validator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/helpers/validatordiag"
+)
+
+var _ tfsdk.AttributeValidator = atLeastSumOfValidator{}
+
+// atLeastSumOfValidator validates that an integer Attribute's value is at least the sum of one
+// or more integer Attributes retrieved via the given path expressions.
+type atLeastSumOfValidator struct {
+	attributesToSumPathExpressions path.Expressions
+}
+
+// Description describes the validation in plain text formatting.
+func (validator atLeastSumOfValidator) Description(_ context.Context) string {
+	var attributePaths []string
+	for _, p := range validator.attributesToSumPathExpressions {
+		attributePaths = append(attributePaths, p.String())
+	}
+
+	return fmt.Sprintf("value must be at least sum of %s", strings.Join(attributePaths, " + "))
+}
+
+// MarkdownDescription describes the validation in Markdown formatting.
+func (validator atLeastSumOfValidator) MarkdownDescription(ctx context.Context) string {
+	return validator.Description(ctx)
+}
+
+// Validate performs the validation.
+func (validator atLeastSumOfValidator) Validate(ctx context.Context, request tfsdk.ValidateAttributeRequest, response *tfsdk.ValidateAttributeResponse) {
+	i, ok := validateInt(ctx, request, response)
+
+	if !ok {
+		return
+	}
+
+	var sumOfAttribs int64
+	var numUnknownAttribsToSum int
```

Comment:
World's smallest nit: Oh my goodness sorry, but since the number of these doesn't matter (anymore 😄 ) should this be a boolean to prevent code readers from thinking the count matters?

Reply:
Even better, I think it can be removed entirely: as soon as an `Unknown` is met, we can return early.

On it.